<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Item_Window</class>
 <widget class="QDialog" name="Add_Item_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-650</x>
     <y>-30</y>
     <width>2031</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 168, 134)</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/AUC Logo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginErrorLabel">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="helloLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="prod_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>Product Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brand_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>Brand Name</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="prod_price">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="category">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="currentText">
    <string>---</string>
   </property>
   <item>
    <property name="text">
     <string>---</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fruit and Vegetables</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meat and Poultry</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dairy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Drinks</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grains</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Desert</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Frozen Food</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="product_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Product Already Exists&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_fields_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*All Fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="stock_count">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Stock</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
